nexus/celery.py

```diff
@@ -37,4 +40,9 @@ def on_configure(self):
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
 
-app.conf.beat_schedule = {}
+app.conf.beat_schedule = {
+    'queue-posts': {
+        'task': 'queue_posts',
+        'schedule': timedelta(minutes=MINUTES_BEFORE_POSTING_ANOTHER_BATCH),
```

Comment:
`MINUTES_BEFORE_POSTING_ANOTHER_BATCH` The variable name doesn't make sense.

Imagine what happens when you add more tasks here and need to schedule it the same way.

So, I think you can do `POST_PUBLISH_CHECK_INTERVAL` or the likes.